coursemapper-kg/recommendation/app/log.py

```
@@ -0,0 +1,66 @@
+        file_handler = TimedRotatingFileHandler(self.log_file, when="midnight")
+        file_handler.setFormatter(self.formatter)
+        return file_handler
+
+    def get_redis_handler(self):
+        redis_handler = RedisHandler(worker_id, redis)
+        redis_handler.setFormatter(self.formatter)
+        return redis_handler
+
+
+class RedisHandler(Handler):
+    def __init__(self, worker_id, redis):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)